java-frontend/src/main/java/org/sonar/java/resolve/Resolve.java

```diff
@@ -505,6 +508,10 @@ private boolean isArgumentsAcceptable(List<JavaType> argTypes, List<JavaType> fo
   }
 
   private boolean isAcceptableType(JavaType arg, JavaType formal, boolean autoboxing) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "isAcceptableType" is 12 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)